replay_server/constants.py

```diff
@@ -0,0 +1,53 @@
+import os
+
+BASE_DIR = os.path.abspath(os.path.join(os.path.dirname(os.path.realpath(__file__)), '..', '..'))
+
+# logging
+LOGGING_LEVEL = int(os.environ.get("LOGGING_LEVEL", 20))  # INFO = 20 DEBUG = 10 ERROR = 40
+ENV = os.environ.get("ENVIRONMENT", "development")  # in production it MUST be "production"
+APP_VERSION = os.environ.get("APP_VERSION", "0.1")  # used for bugsnag logging
+
+# server config
+PORT = int(os.environ.get("PORT", 15000))
+# Timeout step for waiting till next stream will come.
+WAIT_STEP = float(os.environ.get("WAIT_STEP", 1))  # in seconds, it might influence fluent streaming, default is 0.1
+READ_BUFFER_SIZE = 65535 + 1 + 2  # 65535 max tick size + 1 for type + 2 for tick content size
+TICK_COUNT_TIMEOUT = int(os.environ.get("TICK_COUNT_TIMEOUT", 3000))  # 5 minutes * 10 tick per sec
+WRITE_BUFFER_SIZE = int(os.environ.get("WRITE_BUFFER_SIZE", 1024))
+DATABASE_WRITE_WAIT_TIME = float(os.environ.get("DATABASE_WRITE_WAIT_TIME", 10))
+# base for uploading streams
+TEMPORARY_DIR = os.environ.get("TEMPORARY_DIR", "./tmp/")
+# base for uploaded streams
+REPLAY_DIR = os.environ.get("REPLAY_DIR", "./replays/")
+
+# Replay specific
+TERMINATOR = b'\x00'  # null terminator
+
+# actions for putting and getting stream
+PUT_ACTION = b'P'
+GET_ACTION = b'G'
+NEW_LINE = b'/'
+
+
+class RequestType:
+    WRITER = 0  # saving stream
+    READER = 1  # serving stream
```

Comment:
Why not use Python's Enum class and assign "P" and "G" to these?

Reply:
What will give us one more import + one more inheritance? Just less readable code.

Reply:
I'll remove that class and make only `REQUEST_TYPE_WRITER`, `REQUEST_TYPE_READER`.

Reply:
Enum signifies intent, plus it has many convenience goodies - comparison, enum types don't equal raw values, initialization from raw value etc.